test/test_timer.py

```diff
@@ -0,0 +1,25 @@
+from ..openwpm import TaskManager
+from ..openwpm.utilities import db_utils
+from .openwpmtest import OpenWPMTest
+
+
+class TestCommandDuration(OpenWPMTest):
+    def get_config(self, data_dir=""):
+        return self.get_test_config(data_dir)
+
+    def test_command_duration(self):
+        manager_params, browser_params = self.get_config()
+        manager = TaskManager.TaskManager(manager_params, browser_params)
+        manager.get(url="http://www.example.com", sleep=5)
+        manager.close()
+
+        get_command = db_utils.query_db(
```

Comment:
I think you should add a `WHERE` clause to query to make sure you are only getting the get_command

Reply:
Ahh... yes, It means we can also use `WHERE` in this [issue](https://github.com/mozilla/OpenWPM/issues/800#issue-745413888) as well then

Reply:
Done in the latest commit.